src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(firstValue, secondValue) {
+      const resultAdd = firstValue + secondValue;
+
+      return resultAdd;
```

Comment:
Well done! 
As option, you can reduce your code a little bit on rows similar to these.